### ISSUE TITLE
[NavigationView] Talkback does not announce the number of items in the NavigationView

### ISSUE THREAD

**Issue by sixpi** [material-components/material-components-android#678]
**Description:** Talkback does not behave as expected on the navigation drawer - it is including separators between menu item groups in the count of items in the list of items in the navigation drawer. Additionally, the header view automatically adds 1 to the count of items, even if it has no focusable items or if it has more than 1 focusable items.

**Expected behavior:**

I would expect talkback to announce the total number of focusable elements in the navigation drawer, regardless of whether they are menu items or children of the header view(s) (and I suppose footer views as well). I would not expect any automatically inserted separators between menu item groups to be counted. 

**Source code:** 

Minimal sample which adds a navigation drawer with 2 groups and a header containing 3 items is attached. The app will announce 7 items instead of 8 items. Additionally, if `android:focusable="true"` is removed from the items in `menu_header.xml`, then the entire header view is focusable.

[NavigationDrawerDemo.zip](https://github.com/material-components/material-components-android/files/3726429/NavigationDrawerDemo.zip)

**Android API version:** 29, suspect it affects all API levels

**Material Library version:** 1.0.0

**Device:** Pixel 2

To help us triage faster, please check to make sure you are using the [latest version](https://github.com/material-components/material-components-android/releases) of the library.

We also happily accept [pull requests](https://github.com/material-components/material-components-android/pulls).

**Comment by ldjcmu** [comment]
So we looked into this on 1.0.0-beta02 and we no longer announce any item counts. I'm updating the issue to reflect this.

**Comment by chklow** [comment]
We are still seeing navigation view not announcing item counts in 1.4.0

**Comment by gracemichnovicz** [comment]
Wanted to see if there's any update on this / ETA on a fix.
